app/views/liquids/_codesandbox.html.erb

```diff
@@ -1,5 +1,5 @@
 <iframe
   src="<%= "https://codesandbox.io/embed/#{id}#{query}" %>"
   style="width:100%; height:calc(300px + 8vw); border:0; border-radius: 4px; overflow:hidden;"
-  sandbox="allow-same-origin allow-scripts allow-forms allow-top-navigation-by-user-activation">
+  sandbox="allow-same-origin allow-scripts allow-forms allow-popups allow-top-navigation-by-user-activation">
```

Comment:
@garethx If you follow the manual steps, you'll get

```html
<iframe
  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
  src="https://codesandbox.io/embed/react-timeline-demo-1-srpe2?fontsize=14"
  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
  title="react-timeline-demo-1"
></iframe>
```

So I think you should make this
```suggestion
  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin">
```

Reply:
Thanks!

Reply:
Will create a PR for the docs, so it's in sync there too 🙂